<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWidget</class>
 <widget class="QWidget" name="WelcomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="backgroundLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>250</number>
      </property>
      <property name="rightMargin">
       <number>250</number>
      </property>
      <item>
       <spacer name="verticalSpacer_1">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loginBtn">
        <property name="styleSheet">
         <string>
          QPushButton {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #9b59b6, stop:1 #8e44ad);
            border-radius: 20px; color: white; font: bold 20px &quot;微软雅黑&quot;; min-height: 50px;
          }
          QPushButton:hover { background: #a569bd; }
        </string>
        </property>
        <property name="text">
         <string>进入登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="registerBtn">
        <property name="styleSheet">
         <string>
          QPushButton {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #3498db, stop:1 #2980b9);
            border-radius: 20px; color: white; font: bold 20px &quot;微软雅黑&quot;; min-height: 50px;
          }
          QPushButton:hover { background: #5dade2; }
        </string>
        </property>
        <property name="text">
         <string>立即注册</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
